CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 All notable changes to this project will be documented in this file.
 
+## 2.9.9
```

Comment:
shouldn't this include https://github.com/grafana/timestream-datasource/pull/311 ?

Reply:
Oops, done!